### ISSUE TITLE
New local range for stabilizes

### ISSUE THREAD

**Issue by rachhuntley** [geneontology/annotation_extensions#62]
Similar to issue #60 could you please allow use of RNAcentral IDs and IntAct complex IDs with this relation.

Thanks.

**Comment by rachhuntley** [comment]
The local domain would have to change as well to be able to allow this. Currently it is 'regulation of protein stability'. Would it make sense to create a new term that would encompass regulation of protein (GO:0031647), RNA (GO:0043487) and protein complex (GO:0061635) stability terms and then have the parent term as the local domain of 'stabilizes'?

**Comment by ValWood** [comment]
This is a weird term.
defined 
Any process involved in maintaining the structure and integrity of a protein and preventing it from degradation or aggregation.

but regulation of protein degradation isn't even related to this term (it has no descendants)
It might be a bit like this:
https://github.com/geneontology/go-ontology/issues/12294

Stability sounds like the outcome rather than a process... or it could be just badly defined..

**Comment by dosumis** [comment]
From https://github.com/geneontology/annotation_extensions/blob/master/doc/stabilizes.md

Local range is currently:

CHEBI:33697 : ribonucleic acid; GO:0032991 : macromolecular complex; MI:0315 : protein complex; PR:000000001 : protein; SO:0000673 : transcript; SO:0000704 : gene

This looks like it should already do what you want - cover protein, RNA, macromolecular complex.  Maybe we should delete gene? "MI:0315 : protein complex" is probably not needed, but I don't think it does any harm.

> The local domain would have to change as well to be able to allow this. Currently it is 'regulation of protein stability'. Would it make sense to create a new term that would encompass regulation of protein (GO:0031647), RNA (GO:0043487) and protein complex (GO:0061635) stability terms and then have the parent term as the local domain of 'stabilizes'?

Makes sense for this use case - but only because local domain is (rather artificially) restricted to a single term*.  This feels like a bit of an odd amalgam for a GO term.  Can you suggest a name and definition?

\* @tonysawfordebi: Is it possible to extend local domain to allow for multiple terms - just as local range does?

**Comment by dosumis** [comment]
@ValWood - can't find 'regulation of protein degradation'

**Comment by tonysawfordebi** [comment]
Yes, it's possible to make local domain multi-valued, but that would mean quite a bit of upheaval, and it's definitely not a 5 minute job, so, at the moment, it's not something that I would want to do unless there was no alternative.

**Comment by rachhuntley** [comment]
For a grouping term, could you use "regulation of organic substance stability"? There is a term "organic substance transport" which has "protein transport" and "RNA transport" as descendants.

Def: Any process that affects the structure and integrity of an organic substance, altering the likelihood of its degradation or aggregation. An organic substance is a molecular entity that contains carbon.

I think Val must mean protein catabolic process, which is not related to protein destabilization (a child of regulation of protein stability). But I'm not sure if you would expect them to be related. Protein destabilization is defined as "Any process that decreases the stability of a protein, making it more vulnerable to degradative processes or aggregation." It comes under regulation of biological quality.

**Comment by ValWood** [comment]
When  people refer to protein stabilty, in my experience they use this because they don't know whether the process affected is  degradation, or its incorporation into a complex (i.e polymerization).
What do we mean by aggregation here?

I don't know of anything which is involved in  "Any process that decreases the stability of a protein, making it more vulnerable to degradative processes or aggregation."
do you?

GO:0031648 protein destabilization has only 59 experimental annotations.  I looked at the first 4. These all seem to be about protein degradation, autophagy.

Isn't stability just a quality representing the balance between processes of folding and degradation (this is how it is often used). Or is it a real process where a gene  product can  actively destabilise andother protein (I don't see any suggestion of this in the papers I looked at which are used to annotate to this term).

**Comment by ValWood** [comment]
Things like this are phenotypes

"Surprisingly, we found that Bap180 is not essential for viability, although it is required in ovarian follicle cells for normal eggshell development. Bap170 is necessary to stabilize the Bap180 protein, but a mutant form that retains this function is sufficient for both survival and fertility."

If you mutate one protein you decrease the stability of another protein. This isn't a process....its a phenotype....

**Comment by ValWood** [comment]
This term looks sensible.
maintenance of unfolded protein involved in ERAD pathway
but I don't think "protein stability" is a sensible parent for it.

**Comment by dosumis** [comment]
Apparently @tonysawfordebi  has extension of domain to be multi-value on his list to do.  So the proposed change to the relation will soon be technically possible.

**Comment by ValWood** [comment]
It seems that GO term conflates your precise meaning, of 'stabilizing a protein' in an unfolded state, and a much broader  and more general use  where people refer to stability when they measure how l much of a protein they can detect when comparing wt vs mutants.

We probably want terms to describe  the former in biological process but not the latter.

**Comment by rachhuntley** [comment]
I'm not sure about protein stability, but there does seem to be something akin to reducing mRNA stability. miRNAs can induce deadenylation of the poly(A) tail and only when it reaches a length of 20-30nt or less does decay of the transcribed portion of mRNA. 
From http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1449641/ "By analogy to known pathways for mRNA degradation (25), poly(A) tail loss would be expected to facilitate decay of the transcribed portion of targeted mRNAs by exposing either their 5′ ends to cap removal and subsequent degradation by the 5′ exonuclease Xrn1 or their 3′ ends to exonucleolytic degradation by the exosome."

So wouldn't the initial deadenylation be classed as regulation of mRNA stability? However, in these cases I would use the term GO:0098806 deadenylation involved in gene silencing by miRNA, which is a child of mRNA catabolic process!

I should perhaps say why I opened this issue. We are trying to clean up the AE relations used in annotations, there are several annotations using "stability" terms that use has_regulation_target or has_direct_input relations with a gene product in the extension that could possibly use "stabilizes" instead.

**Comment by ValWood** [comment]
Theres something odd about stability. I can't figure out what it is but its an 'outcome' rather than a process. Maybe D O-S can see what the oddness is.

**Comment by dosumis** [comment]
An attempt at resolving the ontological status (may or may not help...):

Stability is not a process. But a process may increase or decrease the stability of a protein, a protein complex, a microtubule, an RNA molecule...

We could define a quality 'stability' and use that to define the relation:

X stabilizes some Y: expandsTO: X positively_regulates some (stability that inheres_in some Y)

This support => the current placement of the relation under has_input .  If the regulation is indirect then Y need not be an input to the stabilising process.

So perhaps:
X stabilizes some Y: expandsTO: X directly_positively_regulates some (stability that inheres_in some Y)

(Note - has_regulation_target doesn't work here, because this is regulation of a quality, not a process).

We can then give 'protein stabilization' the logical definition:

EquivalentTo biological_process that stabilizes some protein.

If we agree that many types of process can do this, then you might want to leave the local_domain very general, that way you'll automatically get folding under 'protein stabilization' any time you make the extension stabilizes{protein identifier}

#### Or alternatively...

Rather than rolling a new relation, we have another option - which would be consistent with the way we usually deal with regulation of qualities in the ontologies:  We can roll terms for 'protein stability', 'RNA stability' etc in [OBA](https://github.com/obophenotype/bio-attribute-ontology).  In the ontology, we could then define 'protein stabilization' EquivalentTo:  biological_process that directly_positively_regulates some OBA:protein stability.  It may be hard to get this strategy to work nicely with annotation extensions though.
